lib/find-view.coffee

```diff
@@ -447,21 +447,27 @@ class FindView extends View
     else
       optionButton.removeClass 'selected'
 
+  anyMarkersAreSelected: =>
+    selections = @model.getEditor()?.getSelections() or []
+    _.any selections, (selection) =>
+      _.any @model.markers or [], (marker) =>
+        _.isEqual(marker.getBufferRange(), selection.getBufferRange())
```

Comment:
I'm not keen on the underscore usage here. If there are a ton of markers this will probably break down. @maxbrunsfeld  @nathansobo is there a nice way to check if any markers are selected?